README.md

```diff
@@ -75,43 +172,42 @@ in two modes:
     $ mb diff --main-branch
     ```
 
-### Walking the dependency graph
-
 The main difference between the above `git diff`s and `mb diff` is the
 dependency graph awareness.
 
-Monobuild will start with the list from git diff, filter it down to known
+Monobuild will start with the list from `git diff`, filter it down to known
 components, and then extend it with all components that depend on any of the
-components in the initial list, including transitive dependencies. The dependency
-graph is built from the `Dependencies` files.
+components in the initial list, including transitive dependencies.
 
-You can validate and print the dependency graph with
+For the resulting "to do" list, `diff` will then build a build schedule using the
+strong dependencies.
 
-```sh
-$ mb validate
+You can print the relevant part of the dependency graph (rather than
+the build schedule) with `--dependencies`
+
+```
+$ mb diff --dependencies
 ```
 
-Like all commands, print also takes `--dependency-files` flag
+Both modes also support DOT output with `--dot`. You can also print
+the entire graph with the affected components with `--dot-highlight`.
 
-```sh
-$ mb validate --dependency-files **/Dependencies
-```
+### Override the manifest matching
+
+If you want to use a different filename for the manifest files, you can do so
+using the global `--manifests` flag.
 
 ### Creating a Makefile
 
 Monobuild can also generate a `Makefile`, that can be used by individual
-component builds to assemble themselves. The make dependency graph is
-the same graph as the one used to decide what should build (except the edges
-are reversed).
+component builds to build their dependencies.
 
 You can generate the makefile with
 
 ```sh
 $ mb makefile
```

Comment:
it was `monobuild` earlier in the readme (there are a couple of examples where it's now `mb`)